nasa-gibs/worldview
# Change chlorophyll a to lower case

## Description

Fixes WV-2878

Changed Chlorophyll A to Chlorophyll a where appropriate. 

## How To Test

1. 'git checkout'
2. 'npm run build && npm start'
3. Go to Add Layers, type in "Chlorophyll", verify that all upper case "A"s have been converted to lower case "a"s, especially in the facets list. In some layer descriptions, the lower case "a" is italicized, which is intentional. 


@nasa-gibs/worldview
